docs/actors/running.md

```diff
@@ -71,6 +71,16 @@ Each run starts with the initial status **READY** and goes through one or more t
 |ABORTING|transitional|Being aborted by user|
 |ABORTED|terminal|Aborted by user|
 
+## [](#aborting-run)Aborting run
+
+Each run in statuses **READY**, **RUNNING**, **TIMING-OUT** can be aborted by user. There are two ways how to abort the run.
+
+* Force abort run - The user immediately aborts the actor. The force abort is the default option.
```

Comment:
@m-murasovs is it `force` abort a good name?

Perhaps we could have just "abort" and "graceful abort"? I don't really feel what is better.

Reply:
Yeah, we can probably go without 'force', 'abort' on its own is strong enough :)